fe/fe-core/src/main/java/org/apache/doris/nereids/properties/PhysicalProperties.java

```diff
@@ -17,35 +17,47 @@
 
 package org.apache.doris.nereids.properties;
 
+import java.util.Objects;
+
 /**
  * Physical properties used in cascades.
- * TODO(wj): Do we need to `PhysicalPropertySpec` Interface like NoisePage?
  */
 public class PhysicalProperties {
+
+    public static PhysicalProperties ANY = new PhysicalProperties();
+
+    public static PhysicalProperties REPLICATED = new PhysicalProperties(DistributionSpecReplicated.INSTANCE);
+
+    public static PhysicalProperties GATHER = new PhysicalProperties(DistributionSpecGather.INSTANCE);
+
     private final OrderSpec orderSpec;
 
     private final DistributionSpec distributionSpec;
 
-    public PhysicalProperties() {
+    private PhysicalProperties() {
         this.orderSpec = new OrderSpec();
-        this.distributionSpec = DistributionSpecAny.getInstance();
+        this.distributionSpec = DistributionSpecAny.INSTANCE;
     }
 
-    public PhysicalProperties(DistributionSpec distributionSpec) {
+    private PhysicalProperties(DistributionSpec distributionSpec) {
         this.distributionSpec = distributionSpec;
         this.orderSpec = new OrderSpec();
     }
 
     public PhysicalProperties(OrderSpec orderSpec) {
         this.orderSpec = orderSpec;
-        this.distributionSpec = DistributionSpecAny.getInstance();
+        this.distributionSpec = DistributionSpecAny.INSTANCE;
     }
 
     public PhysicalProperties(DistributionSpec distributionSpec, OrderSpec orderSpec) {
         this.distributionSpec = distributionSpec;
         this.orderSpec = orderSpec;
     }
 
+    public static PhysicalProperties createHash(DistributionSpec distributionSpec) {
```

Comment:
this function name is `createHash`, so i think the parameter should be `DistributionSpecHash`